Fabric.Identity.API/scripts/Install-Identity-Windows.ps1

```diff
@@ -102,10 +102,27 @@ if((Test-Path $zipPackage))
 }
 
 
-if(!(Test-Prerequisite '*.NET Core*Windows Server Hosting*' 1.1.30327.81))
+if(!(Test-Prerequisite-Exact '*.NET Core*Windows Server Hosting*' 1.1.30327.81))
```

Comment:
What happens if they have an earlier or later version installed? Are we able to halt the script and tell people what they need to do to resolve the issue?

Reply:
The script will fail with an error stating that a different version is installed. I should be able to add a message to the user to uninstall other versions.